aten/src/ATen/native/quantized/cpu/conv_serialization.h

```diff
@@ -0,0 +1,248 @@
+#pragma once
+
+#include <ATen/ATen.h>
+#include <ATen/core/List.h>
+
+#include <ATen/native/quantized/cpu/fbgemm_utils.h>
+#include <ATen/native/quantized/cpu/qnnpack_utils.h>
+
+#include <tuple>
+
+/* Convolution prepacked parameters serialization.
+ *
+ * Files that need to be updated if version changes:
+ * - ATen/native/quantized/cpu/fbgemm_utils.cpp
+ * - jit/passes/onnx/unpack_quantized_weights.cpp
+ *
+ * Version 1
+ *
+ * - Fields:
+ *  1. weight
+ *  2. bias
+ *  3. stride x kSpatialDim
+ *  4. padding x kSpatialDim
+ *  5. dilation x kSpatialDim
+ *  6. groups
+ *
+ * Version 2
+ *
+ * - Fields:
+ *  0. version (string)
+ *  1. list of non-optional tensors
+ *    0: packed parameters (int64_t)
```

Comment:
Can we make this an unspecified integer type, and have the deserializer handle any integer type, then serialize with int8?  That saves a few percent of model size on the model I tested.